packages/cms-cli/commands/auth.js

```diff
@@ -67,8 +88,25 @@ async function authAction(type, options) {
   trackCommandUsage(COMMAND_NAME);
   let configData;
   let updatedConfig;
-  let promptAnswer;
+  let validName;
   switch (authType) {
+    case API_KEY_AUTH_METHOD.value:
```

Comment:
The yargs command description below needs updated to reflect the new option

Reply:
Nice catch. Strange that it seems to work regardless of this value being in the `choices` array.